.github/workflows/ci.yml

```diff
@@ -280,6 +280,8 @@ jobs:
         run: just pnpm-install
       - name: Build Language Plugins
         run: just build-language-plugins
+      - name: Pull Registry
+        run: docker image pull registry:2
```

Comment:
I think we should just use `docker compose` for this yeah?

Reply:
For some reason it was failing to pull on CI, and I have no idea why. Explicitly pulling it worked for some reason. I can remove this and try again in case it was a transient failure, but it was very odd.